lib/__tests__/timing.test.mjs

```diff
@@ -0,0 +1,44 @@
+import process from 'node:process';
+
+import timing from '../timing.mjs';
+
+describe('timing', () => {
+	describe('getListSize()', () => {
+		afterEach(() => {
+			delete process.env.TIMING;
+		});
+
+		it('should return minimum list size when TIMING is not set or has a small value', () => {
+			delete process.env.TIMING;
```

Comment:
Normally you do something like this:

```js
  const OLD_ENV = process.env;

  beforeEach(() => {
    jest.resetModules()
    process.env = { ...OLD_ENV };
  });

  afterEach(() => {
    process.env = OLD_ENV;
  });
```

Reply:
@Mouvedia 
Sorry, I'm late.
I've applied your suggestion in [this commit](https://github.com/stylelint/stylelint/pull/8108/commits/7668318f8c7bf55d048eed374f805ff10f219523).

Reply:
The reason why I suggested this was because of 31328ee.
i.e. you should be able to revert it now

Reply:
I tried to revert that commit, but the test fails.